quickstarts/node-exporter/config.yml

```
@@ -0,0 +1,73 @@
+# Name of the quickstart (required)
+# sets the URL name of the quickstart on public I/O
+name: node-exporter
```

We recently changed this field over to `slug`, could you update this field name?
```suggestion
slug: node-exporter
```

---

@aswanson-nr  Updated. Thanks for the heads up on that one.  Could you hold off on the merge for this one?  I want some additional feedback first.  I will update this PR when I'm ready to go if thats ok with you?